File: src/main/java/com/manson/domain/fed76/PriceCheckResponse.java
package com.manson.domain.fed76;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@EqualsAndHashCode
@ToString
@JsonIgnoreProperties(ignoreUnknown = true)
public class PriceCheckResponse {

    private String name;
    private Integer price;
    private String url;
    private String category;
    private String reviewDescription;
    private String reviewName;
    private String description;
    private String rating;
    private String verdict;

}

File: src/main/kotlin/com/manson/domain/fed76/pricing/PriceCheckResponse.kt
package com.manson.domain.fed76.pricing

import com.manson.AbstractObject

open class PriceCheckResponse : AbstractObject() {

    var name: String = "EMPTY"
    var price: Int = 0
    var review: ReviewResponse = ReviewResponse()
    var timestamp: String = "EMPTY"
    var path: String = "EMPTY"
    var description: String = "EMPTY"

    override fun toString(): String {
        return "PriceCheckResponse(name='$name', price=$price, review=$review, timestamp='$timestamp', path='$path', description='$description')"
    }


}